### ISSUE TITLE
Playback error

### ISSUE THREAD

**Issue by tommyMX** [plexinc-plugins/NBC.bundle#22]
An error occurred trying to play this item.
shaka1001 (Netzwerk)

**Comment by sander1** [comment]
I see "Netzwerk" in your error, which to me sounds German. So: are you in the US? I ask because this channel does not work outside the US.

**Comment by tommyMX** [comment]
Yes gut using vpn usa for it. Other Channels Like the cbs are working on this way. But the nbc Plugin isnt working.

Work this for you?

**Comment by sander1** [comment]
Yes, NBC works for me, at least in Plex Media Player. It's a bit slow when loading the video, but after 15~20 seconds video playback starts just fine.

It could be an issue with your VPN where NBC/Akamai block connections from know VPN provider IPs.
